zenbones.yaml

```diff
@@ -0,0 +1,19 @@
+scheme: "Zenbones"
+author: "mcchrish"
+base00: "#191919"
```

Comment:
Unfortunately, in this repo, I think we're requiring themes to *not* have the leading `#` even though builders support them. We've got some ongoing work to change the format, but that will happen in another repo.

I'd be happy to merge this once those are removed.